src/utils/__tests__/webcrypto.test.js

```diff
@@ -4,6 +4,16 @@ import { digest, getRandomValues } from '../webcrypto';
 jest.mock('js-sha1');
 
 describe('util/webcrypto', () => {
+    beforeEach(() => {
+        // eslint-disable-next-line no-undef
+        Object.defineProperty(globalThis, 'window', {
+            value: {
+                ...window,
+            },
+            writable: true,
+        });
+    });
+
```

Comment:
had to add this because after bumping up to latest, it wasnt allowing the window object to be overridden with properties that are not supported such as `crypto.subtle`